### ISSUE TITLE
Please make the (dark) theme a bit more contrasty

### ISSUE THREAD

**Issue by karolyi** [equinusocio/material-theme#254]
Hey man,

great theme, I'm using the dark version. My only problem is, I'd need more contrast here and there. I'll paste an annotated screenshot about what I exactly mean:

![sshot](https://cloud.githubusercontent.com/assets/987055/8373447/2f292baa-1bed-11e5-9109-ffd1652cfc15.png)

Your efforts are greatly appreciated!

Cheers

**Comment by unlucio** [comment]
just wondering: why do you feel like needing more contrast?
Imoh they're great the way they are: they stay out of the way until you actually need them and focus your eyes on those parts. More contrast would probably feel clutter-ish and increase eyes stress.

**Comment by karolyi** [comment]
That's my problem: I get an eye strain when I want to read them.

If you don't want to modify them, could you please tell me how I can modify them for myself?

**Comment by unlucio** [comment]
I don't actually know, but: have you tried with eye glasses? :P

**Comment by karolyi** [comment]
My eyes are in perfect condition. It's just that I need a more contrasty theme. I might as well switch back to [Monokai-extended](https://github.com/jonschlinkert/sublime-monokai-extended). Until recently, I used that theme. Let me show you what's the difference. That's your theme:
![image](https://cloud.githubusercontent.com/assets/987055/8382908/8d41c5c8-1c36-11e5-8b45-406060cfbff0.png)

And here's Monokai Extended:
![image](https://cloud.githubusercontent.com/assets/987055/8382937/bb61cc32-1c36-11e5-97d1-bdc4623a13a7.png)

**Comment by karolyi** [comment]
Also, the thing with the keyword highlight:
![image](https://cloud.githubusercontent.com/assets/987055/8382960/ee5d4af8-1c36-11e5-8f56-36ecdf9431ad.png)

**Comment by sasindumendis** [comment]
IMHO a slight increase of contrast for comments wouldn't hurt, I am perfectly happy with the contrast levels of the other parts though, like said above, I don't want too many things get in the way at a given time.

**Comment by helgatheviking** [comment]
+1 The comments are definitely difficult to see.

**Comment by equinusocio** [comment]
Sorry for the contrast, but this is a personal preference. I will fix the python issue

**Comment by karolyi** [comment]
Well then, Monokai Extended it is.

**Comment by timbog80** [comment]
@karolyi the syntax theme is perfectly fine contrast wise. quit compaining and if you are having a hard time seeing the words then you 1. need a better monitor or 2. need to get your eyes checked out

**Comment by karolyi** [comment]
So, for those who need an 'upgrade' on the (otherwise brilliant) theme contrastness, you can do it yourselves with a little bit of hacking:

1. Install [PackageResourceViewer](https://packagecontrol.io/packages/PackageResourceViewer)
2. Open the `Material-Theme-Darker.sublime-theme` file within the theme
3. Edit the following lines: [here](https://github.com/karolyi/st3-config/blob/5e983a5ce22424340b31ecc17f9953e6284f9919/Packages/Material%20Theme/Material-Theme-Darker.sublime-theme#L350), [here](https://github.com/karolyi/st3-config/blob/5e983a5ce22424340b31ecc17f9953e6284f9919/Packages/Material%20Theme/Material-Theme-Darker.sublime-theme#L603) [and here](https://github.com/karolyi/st3-config/blob/5e983a5ce22424340b31ecc17f9953e6284f9919/Packages/Material%20Theme/Material-Theme-Darker.sublime-theme#L1127)
4. Save it, you'll see the files, statusbar and tab dark colors lighten up, being more contrasty.

Lightening the comment AND selection color:
1. Open `Material-Theme-Darker.tmTheme` with PackageResourceViewer in the theme
2. Edit the lines [here](https://github.com/karolyi/st3-config/blob/5e983a5ce22424340b31ecc17f9953e6284f9919/Packages/Material%20Theme/schemes/Material-Theme-Darker.tmTheme#L64) and [here](https://github.com/karolyi/st3-config/blob/bf1de407ca0203c11eea833aa307ed9439355a2d/Packages/Material%20Theme/schemes/Material-Theme-Darker.tmTheme#L30)
3. Save the file, and your comment and selection colors will be updated.

The only downside of this approach is that you'll have to redo this edits whenever the original theme refreshes, since Sublime does not have theme inheritance.

You're welcome.

**Comment by equinusocio** [comment]
@karolyi you can keep your change by following this steps.
https://github.com/equinusocio/material-theme/issues/305#issuecomment-120479395

**Comment by karolyi** [comment]
Yeah, I just discovered it ... Does this work for the scheme file too?

**Comment by equinusocio** [comment]
@karolyi It works with all themes file

**Comment by karolyi** [comment]
@equinusocio it seems that the scheme modification doesn't work, only the theme color ones.

**Comment by equinusocio** [comment]
@karolyi You have to rebuild the theme structure, so, the modified schemes must be inside ```schemes\darker\``` called ```Material-Theme-Darker.tmTheme```

**Comment by karolyi** [comment]
@equinusocio nope, doesn't help. I guess this has to do with the fact that the theme file path is explicitly specified in the preferences. Thank you anyways, I'll keep this XML up to date myself, the theme extension was already a great help.

**Comment by sg-s** [comment]
@karolyi thanks for the tip -- i found the statusbar too faint too.

**Comment by timbog80** [comment]
no need for contrast looks good as is, please dont change

**Comment by karolyi** [comment]
@sg-s you're welcome. it seems not everyone turns the brightness totally up on their monitors.

**Comment by equinusocio** [comment]
@karolyi And not everyone turn down the brightness. Sorry. If you want you can make changes to your local installed theme.

**Comment by karolyi** [comment]
@equinusocio which is exactly what I did :)

**Comment by equinusocio** [comment]
@karolyi Nice :)

**Comment by kleinfreund** [comment]
Contrast levels are a pretty crucial aspect for a theme and should be configurable by the user if possible. It would be nice to see accessibility wise to see this in the future.

**Comment by ronaldsuwandi** [comment]
I agree with @karolyi - I think it's still better if the default colour for the comment is a little bit brighter. Yes, user can adjust the colour manually but it's obviously an issue otherwise we won't be having 4 duplicate tickets #711 #290 #393

Perhaps do a vote and see if people do prefer the low contrast comments? I for one find it really hard to read

**Comment by equinusocio** [comment]
@ronaldsuwandi Ask Apple to make a darker Finder ui because the bright one hurt your eyes. This contrast of this theme is balanced to avoid useless distractions (and comments are distractions). 4 issue over 1000 and 420K installation is not common request. Is a personal request, and you can customise the UI by yourself.

Sublime Text doesn't allow custom settings with open values, so it will be never possible allow user to set a custom comments color. I can even make a settings to set "bright" comments, but as i said, 4 issues over 1000 opened issues and 420K installation makes me no thing about that. 

Note: At the moment i'm working on the vsc material theme, since sublime text is not more my favourite editor and i'm the only one that contributed to this project, i will leave this theme as is.

PR are always welcome.

**Comment by ronaldsuwandi** [comment]
> Ask Apple to make a darker Finder ui because the bright one hurt your eyes. 
This is not a relevant comparison, it's about readability, nothing to do with hurting eyes or anything...

> The contrast of this theme is balanced to avoid useless distractions (and comments are distractions until you need them)
Point taken, although I don't fully agree. If you follow Clean Code book suggestion, the codebase will hardly have any comments unless it's for some sort of code optimization. In this case it's not a distraction, rather an important part of the code.

In any case it's just a suggestion and if you don't like it then it's fine. You're the owner after all

**Comment by equinusocio** [comment]
> This is not a relevant comparison, it's about readability, nothing to do with hurting eyes or anything...

It's the same, one ask to increase contrast because it use a low screen luminosity, one ask to increase contrast because his eyes are broken, one ask to increase contrast because it code inside heaven with a sola light over 9000. This theme works well with a standard luminosity and good eyes (if not, you need glasses)

@ronaldsuwandi Yes nobody said that you don't have to write comments. But you need to see comments when you need to read them. If you focus the comments, you can read them easily if you use a normal display luminosity.

**Comment by ronaldsuwandi** [comment]
Sure. My eyes are normal, I can read the comments but it requires more effort than reading the other parts

As for my comment it's mainly because of your hard stance regarding this issue. It's highly unlikely even if a PR is submitted it'll be approved. After all you still need to approve it before merging it.

I won't argue anymore and will just leave it as it is

**Comment by equinusocio** [comment]
>  but it requires more effort than reading the other parts
this is the point. Only if you want to read comments you have to focus on them, otherwise you can easily ignore them.

**Comment by kleinfreund** [comment]
**TL;DR:** Lowering contrast to de-emphasize at the cost of users is bad.

---

> @equinusocio Ask Apple to make a darker Finder ui because the bright one hurt your eyes.

> @equinusocio […] one ask to increase contrast because his eyes are broken […]

You make this discussion needlessly harder by providing these non-sensical comparisons and statements.

You could’ve just said that this is opinionated software and that you do not intend to accept user requests based on reasons you fail to see.

> @equinusocio […] (if not, you need glasses)

You still don’t seem to understand the issue. Glasses cannot correct too low contrast levels. With a perfectly working pair of eyes, working around a too low contrast will still cause your perceptive system to work harder. It is not a choice you can make. The system will try to perceive and it will try everything to make sense of it.

**Comment by equinusocio** [comment]
@kleinfreund tell that to other 418k material theme users.
